proto/kafka_gateway.proto

```
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package kafka.gateway.v1;
+
+option go_package = "kafka-gateway/proto/kafka/gateway/v1;gatewayv1";
+
+import "google/api/annotations.proto";
```

Buf (1.47.2)

import "google/api/annotations.proto": file does not exist

(COMPILE)